### ISSUE TITLE
No Sounds Play At All (End - Start - During work/break)

### ISSUE THREAD

**Issue by awkbari** [YetAnotherPomodoroApp/YAPA-2#98]
Hi my issue is not even one single sound plays before & after and during YAPA-2 working 
! i've tried many soloutions and changing all settings and couldn't solve this problem . 
Please help me to fix this issue . Thanks

**Comment by awkbari** [comment]
alose my speakers and system settings are well and it is only with YAPA-2

**Comment by floatas** [comment]
Hey, can you attach YAPA settings.json file (My Documents\Yapa) ? Can you check volume mixer to make sure YAPA is not muted ?
![image](https://user-images.githubusercontent.com/7883111/174004639-8cfd3717-643c-4c0d-9744-17f3c99cb95c.png)

**Comment by awkbari** [comment]
Hey dude.
[settings.zip](https://github.com/YetAnotherPomodoroApp/YAPA-2/files/8925220/settings.zip)
This is settings.json file and i checked volume mixer too and got no solution.

**Comment by awkbari** [comment]
I tried YAPA-1 and it's ok when playing notifications. I've this problem just with YAPA-2

**Comment by floatas** [comment]
Could you try checking "Play period start\end sounds" in Sounds settings.

**Comment by Lostrick** [comment]
I'm having this issue too
I've been using it for a long time and it's suddenly not playing any sound
I haven't changed any settings and I can't find the app in sound mixer
I also tried resetting the sound mixer but it's not helping
Also tried deleting all the settings file, restarting my laptop but still nor working

**Comment by floatas** [comment]
@Lostrick Can you check if this is enabled ? 
![image](https://user-images.githubusercontent.com/7883111/176600758-4f812fc8-c484-4c80-903b-2acfb0499492.png)

**Comment by Lostrick** [comment]
yes I've checked that of course, and I also tried to deleted all the json settings in documents, start fresh and still no sound

**Comment by floatas** [comment]
Hmm, I'm not even sure how to debug this issue. @awkbari have you managed to fix your sound issue?

@Lostrick  maybe you could try to reinstall the application? Can you reinstall the latest version and if the issue is still here try some older version?

**Comment by ianzhang366** [comment]
had similar problem, the following doc helped
https://support.microsoft.com/en-us/windows/windows-media-player-12-e8f84f54-cd64-865c-2e83-1d8ec121b5b8

**Comment by songxingling1** [comment]
You can try installing Windows Media Player. This is how I make my YAPA-2 sound
